specification/loadtestservice/data-plane/Microsoft.LoadTestService/stable/2022-11-01/loadtestservice.json

```diff
@@ -0,0 +1,3315 @@
+          "type": "string",
+          "readOnly": true
+        },
+        "subscriptionId": {
+          "description": "Subscription Id of the Azure resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "kind": {
+          "description": "Kind of Azure resource type",
+          "type": "string"
+        }
+      }
+    },
+    "CertificateMetadata": {
+      "description": "Certificates are stored in Azure Key Vault. The certificate name and identifier should be provided.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The value of the certificate for respective type",
+          "type": "string"
+        },
+        "type": {
+          "description": "Type of certificate",
+          "enum": [
+            "AKV_CERT_URI"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "CertificateType",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "AKV_CERT_URI",
+                "description": "If the certificate is stored in an Azure Key Vault"
+              }
+            ]
+          }
+        },
+        "name": {
+          "description": "Name of the certificate.",
+          "type": "string"
+        }
+      }
+    },
+    "Error": {
+      "description": "Error from a REST request.",
+      "required": [
+        "code",
+        "message"
+      ],
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "The error code.",
+          "type": "string"
+        },
+        "message": {
+          "description": "The error message.",
+          "type": "string"
+        },
+        "target": {
+          "description": "The error target.",
+          "type": "string"
+        },
+        "details": {
+          "description": "Additional details and inner errors.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Error"
+          }
+        }
+      }
+    },
+    "ErrorDetails": {
+      "description": "Error details if there is any failure in load test run",
+      "type": "object",
+      "properties": {
+        "message": {
+          "description": "Error details in case test run was not successfully run.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ErrorResponseBody": {
+      "description": "The definition of an error object.",
+      "required": [
+        "error"
+      ],
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/Error"
+        }
+      }
+    },
+    "FileUrl": {
+      "description": "FileUrl Model.",
+      "type": "object",
+      "properties": {
+        "url": {
+          "description": "File URL.",
+          "type": "string"
+        },
+        "fileId": {
+          "description": "File unique identifier.",
```

Comment:
Presumably this is the same value as the "fileId" parameter used to create the file. That parameter is described as "Unique name for test file". I think you should use the same description here. I realize that there is potential confusion with "filename" below, but I think it is important to keep descriptions of inputs and outputs consistent.

As an aside, the service rather consistently describes parameters and properties named XXXId as the "name" of XXX. That too is a little confusing but it is so consistent across the service that I think it is best to stick with that.

Reply:
description is changed